### PR TITLE
RDISCROWD-5747: Color Code the Icons in GIGwork Internet Tabs

### PR DESCRIPTION
**Pull request by ciaracian**
*Issue number of the reported bug or feature request: RDISCROWD-5747*

**Describe your changes**
Checks the environment and sends the appropriate favicon directory name to the submodule.

**Testing performed**
Tested locally using Development testing environment variable.

**Additional context**
Supports changes made to [pybossa-default-theme](https://github.com/bloomberg/pybossa-default-theme/pull/413)

Re: https://github.com/bloomberg/pybossa-default-theme/pull/413#event-9038843088

### CODE REVIEW

**Comment by ciaracian** [comment]
> _Issue number of the reported bug or feature request: RDISCROWD-5747_
> 
> **Describe your changes** Checks the environment and sends the appropriate favicon directory name to the submodule.
> 
> **Testing performed** Tested locally using Development testing environment variable.
> 
> **Additional context** Supports changes made to [pybossa-default-theme](https://github.com/bloomberg/pybossa-default-theme/pull/413)

Deep suggested this morning that instead of using a dictionary of platforms-favicon dirs that instead we identify the appropriate favicon dir in the settings file for each platform separately. I will make this change and maybe it will fix the test issue too.

File: pybossa/core.py [hunk 1]
```diff
@@ -689,7 +689,9 @@ def get_wizard_steps(project=None):
             plugins=plugins,
             ldap_enabled=ldap_enabled,
             get_wizard_steps=get_wizard_steps,
-            server_type=app.config['SERVER_TYPE'])
+            server_type=app.config['SERVER_TYPE'],
+            favicon_dir=app.config['FAVICONS']
```
Comment (kbecker42) [inline pybossa/core.py:693]: I believe this change may not be needed. The HTML file can simply read the favicon filename as a string directly from the HTML. In this case, we can close this PR unmerged. Thanks.

Comment (ciaracian) [reply pybossa/core.py]: I confirmed with Deep that in order to access the FAVICON variable value in base.html we need to define it here.

Comment (kbecker42) [reply pybossa/core.py]: This still may not be required. Please see this [example](https://github.com/bloomberg/pybossa-default-theme/blob/main/templates/projects/update.html#L58) where a config variable is accessed from the HTML template. The same could be done for `FAVICON_FILENAME`. Please feel free to test in the code to verify. Thanks.

**Pull request closed without merging**